lib/code_fund_web/views/view_helpers.ex

```diff
@@ -35,7 +35,9 @@ defmodule CodeFundWeb.ViewHelpers do
       "Mar 3, 2016"
 
   """
-  def pretty_date(%NaiveDateTime{} = date, format \\ "%b %-d, %Y") do
+  def pretty_date(date, format \\ "%b %-d, %Y")
```

Comment:
Thanks for fixing this. I wasn't really understanding it before.

Reply:
You bet, here's some more on that if you're interested https://blog.robphoenix.com/elixir/notes-on-elixir-bodyless-functions/